packages/commonwealth/scripts/resizeImages.ts

```diff
@@ -3,20 +3,19 @@
 // this script will resize the images in the Chains.icon_url that are larger than 1000x1000.
```

Comment:
I don't think we need this script anymore. @kurtisassad can you confirm?

Reply:
I kept it just in case, but we can archive this for sure.

Reply:
Yep we can remove this one